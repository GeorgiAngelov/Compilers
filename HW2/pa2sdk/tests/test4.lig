/* Test of a non-token. */

function f () : int {
      return 3 * 5 ? 3;
}

