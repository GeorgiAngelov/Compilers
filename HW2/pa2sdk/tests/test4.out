syntax error
Genuine Liger: no
